### PR TITLE
Fix status indicator overlapping

### LINKED ISSUE #3136: Availability icon overlaps user status
When you add Nextcloud users to your event and they are online, there will be a status indicator in the bottom left of their avatar. This is also where we overlay the avatar with the availability circle.

Bildschirmfoto von 2021-06-01 17-33-34

---

The overlapping status circle vs the participation circle are not ideal. We might want to move the participation icon to the left? cc @nimisha-vijay

_Originally posted by @ChristophWurst in https://github.com/nextcloud/calendar/issues/3112#issuecomment-852135061_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #3136 
- [x] Change icon position
- [x] Add status text

### CODE REVIEW

**Comment by GretaD** [comment]
right now it looks like this.
indicator_status

Now that we added the text to read the status, should we remove the popover that elaborates what the status is?  @ChristophWurst @tcitworld

**Comment by tcitworld** [comment]
Yes, I think we should remove it, since it was only here to add some context to the avatar indicator which is now gone.

However we need to make sure every information from these popovers is in the subline.

**Comment by GretaD** [comment]
@tcitworld @ChristophWurst  please have a look

status_indicator

**Comment by tcitworld** [comment]
Let's rephrase `%s did not respond to your invitation yet.` in `%s did not respond yet`, as the name might be taking too much space. How is ellipsis handled?

**Comment by ChristophWurst** [comment]
*yet* implies past perfect IIRC, for correct grammar it has to be *hasn't responded yet*

**Review by st3iny — COMMENTED** [review]
Looks good overall and works.

I have one little remark about the design: How about removing the usernames and punctuation from the labels? Currently, the names are shown twice right above each other.

#### Compare the labels of admin and user:
Screenshot 2021-06-11 at 11-47-40 June 2021 - Calendar - Nextcloud

**Comment by tcitworld** [comment]
I like your suggestion @st3iny but it only works for languages that match English order of words (with the subject on the beginning of the sentence).

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 1]
```diff
@@ -93,15 +93,15 @@ export default {
...
+					label: t('calendar', 'Invitation accepted.', {
 						name: this.commonName,
 					}),
 				}
 			}
 			if (this.participationStatus === 'ACCEPTED' && !this.isViewedByOrganizer) {
 				return {
 					class: ['accepted', 'icon', 'icon-checkmark-white'],
-					label: t('calendar', '{name} accepted {organizerName}\'s invitation.', {
+					label: t('calendar', 'accepted {organizerName}\'s invitation.', {
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:104]:
```suggestion
					label: t('calendar', 'Accepted {organizerName}\'s invitation.', {
```

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 2]
```diff
@@ -143,14 +143,14 @@ export default {
...
-					label: t('calendar', '{name} hasn\'t responded yet.', {
+					label: t('calendar', 'Invitation sent', {
 						name: this.commonName,
 					}),
 				}
 			} else {
 				return {
 					class: ['no-response', 'icon', 'icon-invitees-no-response-white'],
-					label: t('calendar', '{name} did not respond to {organizerName}\'s invitation yet.', {
+					label: t('calendar', 'did not respond to {organizerName}\'s invitation yet.', {
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:153]:
```suggestion
					label: t('calendar', 'Has not respond to {organizerName}\'s invitation yet.', {
```

Comment (GretaD) [reply src/components/Editor/Invitees/AvatarParticipationStatus.vue:153]: damn, you;re fast, i was about to write what do you think about the string that has the organizer's name and if its alright to keep it like this. But i guess you just approved :P

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 3]
```diff
@@ -68,84 +71,110 @@ export default {
...
+				}
 			}
 			if (this.isResource && this.participationStatus === 'DECLINED') {
-				return this.$t('calendar', '{name} is not available.', {
-					name: this.commonName,
-				})
+				return {
+					class: ['declined', 'icon', 'icon-close-white'],
+					label: t('calendar', 'Not available.', {
+						name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:88]: there is no more placeholder `name`

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 4]
```diff
@@ -68,84 +71,110 @@ export default {
...
 			}
 
 			if (this.participationStatus === 'ACCEPTED' && this.isViewedByOrganizer) {
-				return this.$t('calendar', '{name} accepted your invitation.', {
-					name: this.commonName,
-				})
+				return {
+					class: ['accepted', 'icon', 'icon-checkmark-white'],
+					label: t('calendar', 'Invitation accepted.', {
+						name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:97]: same

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 5]
```diff
@@ -68,84 +71,110 @@ export default {
...
 			}
 			if (this.participationStatus === 'ACCEPTED' && !this.isViewedByOrganizer) {
-				return this.$t('calendar', '{name} accepted {organizerName}\'s invitation.', {
-					name: this.commonName,
-					organizerName: this.organizerDisplayName,
-				})
+				return {
+					class: ['accepted', 'icon', 'icon-checkmark-white'],
+					label: t('calendar', 'Accepted {organizerName}\'s invitation.', {
+						name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:105]: same

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 6]
```diff
@@ -68,84 +71,110 @@ export default {
...
 			}
 
 			if (this.participationStatus === 'DECLINED' && this.isViewedByOrganizer) {
-				return this.$t('calendar', '{name} declined your invitation.', {
-					name: this.commonName,
-				})
+				return {
+					class: ['declined', 'icon', 'icon-close-white'],
+					label: t('calendar', 'Invitation declined.', {
+						name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:115]: same

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 7]
```diff
@@ -68,84 +71,110 @@ export default {
...
 			}
 			if (this.participationStatus === 'DECLINED' && !this.isViewedByOrganizer) {
-				return this.$t('calendar', '{name} declined {organizerName}\'s invitation.', {
-					name: this.commonName,
-					organizerName: this.organizerDisplayName,
-				})
+				return {
+					class: ['declined', 'icon', 'icon-close-white'],
+					label: t('calendar', 'Declined {organizerName}\'s invitation.', {
+						name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:123]: same

File: src/components/Editor/Invitees/AvatarParticipationStatus.vue [hunk 8]
```diff
@@ -68,84 +71,110 @@ export default {
...
+						organizerName: this.organizerDisplayName,
+					}),
+				}
 			}
 
 			if (this.participationStatus === 'DELEGATED') {
-				return this.$t('calendar', '{name} has delegated their invitation.', {
+				return this.$t('calendar', 'Invitation is delegated.', {
 					name: this.commonName,
```
Comment (ChristophWurst) [inline src/components/Editor/Invitees/AvatarParticipationStatus.vue:131]: same (and I guess a few more times :wink: )

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Review by tcitworld — APPROVED** [review]
Squash before merge

**Pull request merged**